migrations/applied_migrations

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+echo ""
+echo "Applying migration ClaimPeriodStart"
+
```

Comment:
what's this file for?

Reply:
its something for Scaffold and we can ignore this - it wont take part in the real business logic

Reply:
we need to run `./migrate.sh`  each time we create a new brand new page,  the changes made during migration will be logged in this file

Reply:
every controller will require an entry in here?